tsconfig.json

```
@@ -0,0 +1,29 @@
+{
+  "compilerOptions": {
+    "target": "esnext",
```

I think it would be convenient to have a link to the page that has descriptions of all these options at hand:
```suggestion
{
  // Configuration reference: https://www.typescriptlang.org/tsconfig
  "compilerOptions": {
    "target": "esnext",
```